src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,86 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your alert coverage by implementing the following recommendations. 
+
+Read on to learn the best practices for:
+- [Policies](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#policy-practices)
```

Comment:
Also, the order of sections in this doc feels a bit weird to me.

I'll need to think a bit more on this...

Reply:
We talked about this over Slack